docs/index.md

```diff
@@ -226,3 +231,43 @@ Within the OpenShift organization we use kraken to perform chaos testing through
         iii. This test can be seen here: https://github.com/openshift/svt/tree/master/reliability-v2
 
     3. We are starting to add in test cases that perform chaos testing during an upgrade (not many iterations of this have been completed).
+
+### Using kraken as part of a tekton pipeline
```

Comment:
docs/installation.md is the right place for the installation content like you added, we might want to remove the pointers from here given that the guide encapsulates best practices and how folks are using the framework in their environment. It would be great if we can info on how we are leveraging Kraken and the your use case under Chaos testing in Practice section. Thoughts?

Reply:
If you need some change, refactoring or new submission, tell me what to change or improve to make it usable for the majority

Reply:
@startxfr thoughts on having a separate document with all the content - something along the lines of https://github.com/chaos-mesh/chaos-mesh/blob/master/ADOPTERS.md and tagging it in the chaos-testing guide as a additional bullet point under "chaos-testing-in-practice" section?